lib/lambda.js

```diff
@@ -705,6 +705,14 @@ function elasticApmAwsLambda(agent) {
       // Look for trace-context info in headers or messageAttributes.
       let traceparent;
       let tracestate;
+      if (
+        triggerType === TRIGGER_GENERIC &&
+        event.traceparent &&
+        event.tracestate
+      ) {
+        traceparent = event.traceparent;
+        tracestate = event.tracestate;
+      }
```

Comment:
@fdaciuk Thanks for the PR. 

Can you show some simple Lambda function code that uses this? That would help me understand a bit better.
Does your code that is doing the Lambda Invoke API call (https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html) manually pass in `traceparent` and `tracestate` in the payload?

Reply:
Hi @trentm. Exactly! I don't know if there is better way to do this, but that update worked for my use case.

I'm invoking the lambda like this:

```js
  const command = new InvokeCommand({
    FunctionName: "my-lambda",
    InvocationType: "RequestResponse",
    Payload: new TextEncoder().encode(JSON.stringify({ traceparent, tracestate })),
  })
```

Reply:
There isn't a current way to do this ... except possibly Amazon's X-Ray support might enable making the connection. We aren't currently doing anything with X-Ray in our Lambda support.

Providing some way for this manual distributed tracing in Lambda is interesting. I like the idea.

I wonder about instead using `ClientContext` in the [InvokeCommand](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-lambda/classes/invokecommand.html) call.   Theoretically, if the APM agent automatically instrumented `@aws-sdk/client-lambda` -- which I'm guessing you are using here -- then we could (perhaps optionally) support automatically adding the trace-context info (`traceparent` et al) to the ClientContext or Payload.

I'm not sure if `ClientContext` and `Payload`, IIUC, are unstructured. Or does the provided `ClientContext` need to be a JSON object that can be added to the `context` object passed to the Lambda function?  If these are unstructured, in general, then it would be a hard sell to have a default feature added to the APM agent that tried to impose some structure on it.

Reply:
> I'm not sure if ClientContext and Payload, IIUC, are unstructured. Or does the provided ClientContext need to be a JSON object that can be added to the context object passed to the Lambda function?

That's a good question. I'm not sure, but love the idea this to be automatic. We're creating a library on our company to abstract the use of `InvokeCommand` and inject `traceparent` and `tracestate` into Payload, so if we have an option to do this from `elastic-apm-node`, even manually, would be great =)

Reply:
Just a heads up that I won't be able to get to this soon. :/

Reply:
No problem. For now, we're using a forked version of this repo with these updates. I hope it's not a problem until you decide whether or not it's a valuable feature in this lib =)